packages/cli/src/services/checkly-config-loader.ts

```diff
@@ -6,17 +6,13 @@ import { Session } from '../constructs'
 import { Construct } from '../constructs/construct'
 import type { Region } from '..'
 import { ReporterType } from '../reporters/reporter'
-import { BrowserPlaywrightDefaults } from '../constructs/browser-defaults'
 import * as fs from 'fs'
+import { PlaywrightConfig } from '../constructs/playwright-config'
 
 export type CheckConfigDefaults = Pick<CheckProps, 'activated' | 'muted' | 'doubleCheck'
```

Comment:
I think `playwrightConfig` should be here, right?

Reply:
Playwright config is not part of the CheckProps to avoid setting it on all the checks.